pkg/devices/cpu_power_meter.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2025.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package devices
```

Comment:
NIT: Are we calling it `devices` or `meters`? the commit says `sensors`. 
Also keeping the package name singular sounds better just like all (or most) go std packages.

Reply:
the name is kept as per design doc discussion.
suggest the new name, i will update it.